frontend/src/metabase/admin/datamodel/components/database/ColumnItem.jsx

```diff
@@ -210,7 +210,7 @@ export class SpecialTypeAndTargetPicker extends Component {
     );
 
     return (
-      <div>
+      <div className={cx(selectSeparator ? "flex align-center" : null)}>
```

Comment:
Because we're using this component on both the table metadata view as well as in the field settings page, this looks to see whether or not the chevron field separator is present or not, and if so, makes the selects flow in one line.